### PR TITLE
fix: configure webpack 4

### PR DESCRIPTION
**Pull request by KnisterPeter**
To have webpack 4 configured correctly we need to set
the webpack mode.

### CODE REVIEW

File: scripts/make-webpack-config.js [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 const path = require('path');
 const webpack = require('webpack');
+const webpackVersion = require('webpack/package.json').version;
```
Comment (sapegin) [inline scripts/make-webpack-config.js]: We have the `getWebpackVersion` function for that.

File: scripts/make-webpack-config.js [hunk 2]
```diff
@@ -50,6 +51,9 @@ module.exports = function(config, env) {
 			hints: false,
 		},
 	};
+	if (parseInt(webpackVersion[0], 10) >= 4) {
```
Comment (sapegin) [inline scripts/make-webpack-config.js]: You need an empty line before if.

**Comment by KnisterPeter** [comment]
@sapegin Thanks for your hints. Updated accordingly.

**Comment by sapegin** [comment]
Thanks, merged!

**Pull request merged**